source/release-notes/3-1-6.rst

```diff
@@ -0,0 +1,35 @@
+.. index:: Crafter CMS 3.1.6 Release Notes
+
+-----------------
+Crafter CMS 3.1.6
+-----------------
+
+^^^^^^^^^^^^
+New Features
+^^^^^^^^^^^^
+
+* Added workflow / publishing support for S3 bucket storage. 
+* Added support for Dashboard APIs that enables content retrieval based on content and metadata criteria defined by users.
+* Added a new video picker control for mediaconvert data source
+* Added support for cache warming configuration per site
+
+
+^^^^^^^
+Updates
+^^^^^^^
+
+* Added configurable script support to TinyMCE 5 form control
+* Added a property for checking if a site is in staging
+* Added video center blueprint enhancements
```

Comment:
Should this be part of Crafter CMS release notes? I think this should go into that BP (if/when we end up with RNs for those external products).